docker/docker-compose.yml

```diff
@@ -10,18 +10,31 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
     volumes:
       - postgres_data:/var/lib/postgresql/data
-  web:
+    networks:
+      - service_customer_network
+  customer-service:
     build:
       context: ../
       dockerfile: docker/Dockerfile
     restart: always
     depends_on:
-      - db
+      - customer-db
     env_file:
       - .env
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_HOST: ${POSTGRES_HOST}
+      POSTGRES_PORT: ${POSTGRES_PORT}
+      POSTGRES_NAME: ${POSTGRES_NAME}
     volumes:
       - ../customers_manager:/app
```

Comment:
create this as an external volume, the same way as at the 34. line; https://docs.docker.com/reference/compose-file/volumes/